### PR TITLE
Fix the Helm chart's image name

### PR DESCRIPTION
**Pull request by haoxins**

### CODE REVIEW

File: helm-chart/kubray-operator/values.yaml [hunk 1]
```diff
@@ -5,8 +5,8 @@
 replicaCount: 1
 
 image:
-  repository: docker.io/library/controller
-  tag: latest
+  repository: kuberay/operator
```
Comment (haoxins) [inline helm-chart/kubray-operator/values.yaml:8]: Maybe `rayproject/kuberay-operator` is the correct one?

Comment (Jeffwan) [reply helm-chart/kubray-operator/values.yaml:8]: `rayproject/kuberay-operator` is the mirror of `kuberay/operator`, for the image management, please check https://github.com/ray-project/ray/pull/21086#discussion_r788291269 for details. 

`rayproject/kuberay-operator` is recently sync to `rayproject` namespace for autoscalign project. We will need a better way to sync the images in the future

Comment (Jeffwan) [reply helm-chart/kubray-operator/values.yaml:8]: I would suggest to use `kuberay/operator` at this moment.

**Comment by Jeffwan** [comment]
Thanks for the fix. The change looks good to me

**Review by Jeffwan — APPROVED** [review]

**Pull request merged**